src/handlers/positions/handler.js

```diff
@@ -149,7 +149,8 @@ const positions = async (error, messages) => {
       await Utility.produceGeneralMessage(TransferEventType.NOTIFICATION, TransferEventAction.COMMIT, message.value, Utility.ENUMS.STATE.SUCCESS, transferId)
       histTimerEnd({ success: true, fspId: Config.INSTRUMENTATION_METRICS_LABELS.fspId })
       return true
-    } else if (message.value.metadata.event.type === TransferEventType.POSITION && message.value.metadata.event.action === TransferEventAction.REJECT) {
+    } else if (message.value.metadata.event.type === TransferEventType.POSITION &&
+      (message.value.metadata.event.action === TransferEventAction.REJECT || message.value.metadata.event.action === TransferEventAction.ABORT)) {
       Logger.info('PositionHandler::positions::reject')
```

Comment:
Should this logger not be updated to reflect whether this is either a `abort` or `reject`?

Reply:
agree